great_expectations/core/util.py

```diff
@@ -864,11 +862,20 @@ def get_or_create_spark_session(
         if spark_session.sparkContext._jsc.sc().isStopped():
             raise ValueError("SparkContext stopped unexpectedly.")
 
-    except AttributeError:
+    # TODO: <Alex>ALEX</Alex>
+    # except AttributeError:
+    #     logger.error(
+    #         "Unable to load spark context; install optional spark dependency for support."
+    #     )
+    #     spark_session = None
+    # TODO: <Alex>ALEX</Alex>
+    # TODO: <Alex>ALEX</Alex>
```

Comment:
Are these comments necessary? Can we remove them?

Reply:
@cdkini This was debug -- deleting (in next commit).